docs/proposals/20200310-crd-spec-configurability.md

```diff
@@ -40,7 +39,7 @@ superseded-by:
 
 ## Summary
 
-Our current CRD spec has a limited number of exposed properties for users to configure which restricts supported use cases. This KEP proposes to add an override section as an additional way for users to configure their cluster. Users will be able to configure any field of the StatefulSet and client Service through editing the spec directly through this change.
+Our current CRD spec has a limited number of exposed properties for users to configure which restricts supported use cases. This KEP proposes to add an override section as an additional way for users to configure their cluster. Users will be able to configure any field of the StatefulSet and client Service through editing the spec directly through this change
```

Comment:
Nitpick: The change here is the full-stop at the end of the line has been removed. Is that intentional and desired?

Reply:
Updated and moved this doc to `proposal/implemented`